js/screens/HomeScreen.js

```diff
@@ -132,16 +133,15 @@ class HomeScreen extends React.Component {
           console.log(e);
 
           if (e === 'dupe site') {
-            Alert.alert(`${term} already exists`);
+            Alert.alert(Localization('term_exists', {term}));
```

Comment:
Why use the custom `Localization` instead of `I18n.t`? `I18n` might come in handy for other stuff as well, like formatting numbers or dates.